docs/guide/learning/faq.md

```diff
@@ -8,591 +8,47 @@ Answers may contain links to other Vue community pages, external articles and bl
-
-<useful-links>
-<useful-links-section title="Official">
-
-* [Scoped CSS](https://vue-loader.vuejs.org/guide/scoped-css.html)
-
-</useful-links-section>
-</useful-links>
-
-### Can I mix both scoped and global styles in one file
-
-Yes, you can mix them, just define a second style tag.
- 
-<useful-links>
-<useful-links-section title="Official">
-
- * [Mixing Local and Global Styles](https://vue-loader.vuejs.org/guide/scoped-css.html#mixing-local-and-global-styles)
-
-</useful-links-section>
-</useful-links>
-
-### What is the difference between scope styles and css modules
-
-Scoped styles automatically change the css rules to match the auto generated data-attributes on each element that is being styled. This is done without any extra effort from the developer.
-
-CSS Modules allow for, again, uniquely generated classes, that are accessible from within the Vue Template, via a special `$style` property. One major benefit of Modules is that you can export css variables to JavaScript.
-
-<useful-links>
-<useful-links-section title="Tutorials">
-
-* [Vue.js - Scoped Styles vs CSS Modules](https://www.netguru.com/codestories/vue.js-scoped-styles-vs-css-modules)
-
-</useful-links-section>
-</useful-links>
-
-### How can I target children with scoped styles
-
-Using the `deep` selector in your sass files.
-
-<useful-links>
-<useful-links-section title="Tutorial">
-
-[How to Target Child Components with Scoped CSS in Vue](https://bambielli.com/til/2018-08-19-how-to-target-child-components-with-scoped-css-in-vue/#)
-
-</useful-links-section>
-</useful-links>
-
-
-### How to do CSS in JS with Vue
-
-The most popular library for Vue is [vue-styled-components](https://github.com/styled-components/vue-styled-components), which is a port of the original `styled-components` library.
+<!--
+  Contributors:
+
+    The FAQs are now stored outsode of the docs/ directory, in faqs/
+
+    FAQS:   https://github.com/dobromir-hristov/vuecommunity/tree/master/faqs
+    README: https://github.com/dobromir-hristov/vuecommunity/tree/master/faqs/README.md
+-->
+
+<faq-category v-for="(faqs, categoryName) in faqsByCategory" :name="categoryName">
+  <faq-item v-for="faq in faqs" v-bind="faq" />
+</faq-category>
+
+<script>
+export default {
+  computed: {
+    faqsByCategory() {
+      return (this.$page.faqData || [])
+        .reduce((categories, page) => {
+          const category = page.category || 'Uncategorised'
+          if ( ! categories[category]) {
+            categories[category] = []
+          }
+          categories[category].push(page)
+          return categories
+        }, {})
+    }
+  },
+}
+</script>
+
+<style>
+/*
+  HACK: Without this the headings are hidden behind the navigation, this 
+        snippet is taken from the live site. It has something to do with a 
+        `.custom` CSS class. I don't understand it. There must be a better way?
+*/
+h1, h2, h3, h4, h5, h6 {
```

Comment:
This is a global style, it will override all others.

Reply:
Understood. Can't figure out how to fix this issue, it doesn't quite make sense to me. Will come back to it.

Reply:
This one has also been taken care of.